features/makefiles/hellolib/conanfile.py

```diff
@@ -0,0 +1,27 @@
+from conans import ConanFile, AutoToolsBuildEnvironment
+from conans import tools
+
+
+class HelloConan(ConanFile):
+    name = "hello"
+    version = "0.1"
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake"
+    exports_sources = "src/*"
+
+    def build_requirements(self):
+        if self.settings.os == "Windows":
+            self.build_requires("make/4.2.1")
```

Comment:
This was added to make the example build on windows but maybe is too complex for the example in the docs